xbmc/addons/PVRClient.cpp

```diff
@@ -393,6 +393,7 @@ void CPVRClient::WriteEpgTag(const EPG::CConstEpgInfoTagPtr &tag, EPG_TAG &pvrTa
   pvrTag.strIMDBNumber = tag->IMDBNumber().c_str();
   pvrTag.strEpisodeName = tag->EpisodeName().c_str();
   pvrTag.strIconPath = tag->Icon().c_str();
+  pvrTag.iChannelNumber = tag->PVRChannelNumber();
```

Comment:
Ii is related to his PR but the functions was introduced in the first commit. iChannelNumber was missing in the mapping.